### PR TITLE
doc changes

### PR DESCRIPTION
**Pull request by yahu1031**
**- What I did**
- Still working

**- How I did it**
- writing every important function/method implementation.

**- How to verify it**
- check the readme.md

**- Description for the changelog**

> ***Do not merge this PR***

### CODE REVIEW

File: at_client/README.md [hunk 1]
```diff
@@ -1,8 +1,103 @@
-<img src="https://atsign.dev/assets/img/@developersmall.png?sanitize=true">
+<img width=250px src="https://atsign.dev/assets/img/@platform_logo_grey.svg?sanitize=true">
 
-### Now for some internet optimism.
+## Now for some internet optimism.
 
-[![Pub Package](https://img.shields.io/pub/v/at_client)](https://pub.dev/packages/at_client)
+[![pub package](https://img.shields.io/pub/v/at_client_mobile)](https://pub.dev/packages/at_client_mobile) [![pub points](https://badges.bar/at_client_mobile/pub%20points)](https://pub.dev/packages/at_client_mobile/score) [![build status](https://github.com/atsign-foundation/at_client_sdk/actions/workflows/at_client_sdk.yaml/badge.svg?branch=trunk)](https://github.com/atsign-foundation/at_client_sdk/actions/workflows/at_client_sdk.yaml) [![gitHub license](https://img.shields.io/badge/license-BSD3-blue.svg)](./LICENSE)
```
Comment (sarika01) [inline at_client/README.md]: The README is for at_client, but the links are for at_client_mobile.

File: at_client/README.md [hunk 2]
```diff
@@ -1,8 +1,103 @@
...
+        ..namespace = '.my_namespace'
+        ..hiveStoragePath = appSupportDir.path
+        ..commitLogPath = appSupportDirdir.path
+        ..isLocalStoreRequired = true;
+```
+
+- These `preferences` are used for your application.
+
+```dart
+AtClientManager currentAtSign = await AtClientManager.getInstance().setCurrentAtSign(atSign, AtEnv.appNamespace, preferences);
```
Comment (sarika01) [inline at_client/README.md]: Variable name "currentAtSign" is incorrect as the call is returning AtClientManager.

File: at_client/README.md [hunk 3]
```diff
@@ -1,8 +1,103 @@
...
+
+```dart
+bool isDeleted = await atClientInstance.delete(atKey);
+print(isDeleted); // true if deleted
+```
+
+- Sync the data to the server using the `sync()` method if needed.
+
+```dart
+late SyncService syncService;
```
Comment (sarika01) [inline at_client/README.md]: If syncservice is declared late, when does it get initialised?

File: at_client/README.md [hunk 4]
```diff
@@ -1,8 +1,104 @@
...
+          ..sharedWith='@bob'
+          ..metadata=metaData
+        ..namespace = '.myApp';
+String value = (await atClientInstance.get(atKey)).value;
+OperationEnum operation = OperationEnum.update;
+bool isNotified = await atClientManagerInstance.notify(atKey, value, operation);
+print(isNotified); // true if notified
+```
+
+AtClient has many more methods that are exposed. Please refer to the [atsign docs](https://atsign.dev/docs/overview/) for more details.
```
Comment (yahu1031) [inline at_client/README.md]: this is just a demo link, this will be replaced with the other link on release time

File: at_client/README.md (hunk 3 above)
Comment (yahu1031) [reply at_client/README.md]: check that now :)

**Review by cpswan — APPROVED** [review]

**Comment by cpswan** [comment]
Are we going to add an example.md, and if so should some of the usage info move there?

**Comment by yahu1031** [comment]
> Are we going to add an example.md, and if so should some of the usage info move there?

@cpswan I didn't understand what example.md is and where we gonna add it. Please LMK where and how we are planning that.

**Comment by cpswan** [comment]
@yahu1031 please connect with @kalluriramkumar as he's working on something similar with at_lookup

**Review by CynthiaPeter — APPROVED** [review]

**Comment by cpswan** [comment]
@yahu1031 can you please rebase this so that the tests pass

**Comment by yahu1031** [comment]
Review needed for ee407fe commit

File: at_client_mobile/example/README.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Demo application
+
+An example of at_client package usage. To learn more about at_client, checkout [atsign.dev](https://atsign.dev/docs/overview/).
```
Comment (cpswan) [inline at_client_mobile/example/README.md]: There should be more explanation of the demo app here.

**Review by cpswan — DISMISSED** [review]
It's good to see the tests passing again :)

This has turned into a docs change, a refactor AND an example addition, so lots going on here.

Looks like we're almost there though - it just needs a better explanation of the example app.

File: at_client_mobile/README.md [hunk 6]
```diff
@@ -1,8 +1,121 @@
...
+    await _keyChainManager.resetAtSignFromKeychain(atsign);
+}
+```
+
+- Get `AtClientService` instance to manage your atsigns.
+
+```dart
+Map<String?, AtClientService> atClientServiceMap = {};
+
+OnboardingWidgetService().onboarding(
```
Comment (sarika01) [inline at_client_mobile/README.md:97]: This piece of code does not give details of any context. Like where it came from.

**Review by murali-shris — APPROVED** [review]

**Review by sitaram-kalluri — APPROVED** [review]

**Pull request merged**